e2e/components/login/login.po.js

```diff
@@ -14,10 +19,47 @@ class LoginComponent {
         return $(loginElements.userName);
     }
 
+    get passwordInput () {
+        return $(loginElements.password);
+    }
+
+    get loginButton () {
+        return $(loginElements.login);
+    }
+
+    get logoutButton () {
+        return $(loginElements.logout);
+    }
+
     openLoginComponent () {
         this.toggleLoginComponent.click();
         return this;
     }
+
+    loginAsACB () {
+        if (this.usernameInput.isDisplayed() === false) {
```

Comment:
Checking a variable against a boolean value inside an `if` test is redundant. Should just `if` the variable (with or without a `!` as necessary

```suggestion
        if (!this.usernameInput.isDisplayed()) {
```

Reply:
Fixed

Reply:
The comment says "fixed", but I don't see a change in the code. Am I missing something? There are a few other comments like that through the PR